.github/actions/release-package/index.mjs

```diff
@@ -5,14 +5,15 @@ import { existsSync, readFileSync, writeFileSync } from 'fs';
 const inputs = {
   path: process.env.INPUT_PATH,
   suffix: process.env.INPUT_SUFFIX,
+  publishPackages: process.env.PUBLISH_PACKAGES?.split(',').map((pkg) => pkg.trim()),
 };
 
-// The main packags should publish to next, and dev forks to next-dev
+// The main branch should publish to next, and dev forks to next-dev
 const branchName = process.env.GITHUB_REF_TYPE === 'branch' ? process.env.GITHUB_REF_NAME : '';
 const publishTag = branchName.startsWith('dev-v3-') ? branchName : 'next';
 
 const subPackages = {
-  'components': [
+  components: [
     'lib/components',
     'lib/style-dictionary',
     'lib/components-themeable',
```

Comment:
Currently we have this hard-coded. With the change, it will move into workflow definition of the components repo

Reply:
Do we have a plan to move the definition to the components repo? I'm asking to understand, if this is going to be added to the component repo or we'd have this option which will then not be used.

Reply:
Yes, this the the plan
1. Merge this PR
2. Open PR for the sub packages to update their local workflow configs
3. When all PRs merged, open another PR here to clean up the unused parts